Sources/SQLKit/Query/SQLReturning.swift

```diff
@@ -0,0 +1,14 @@
+/// `RETURNING ...` statement.
+///
+public struct SQLReturning: SQLExpression {
+    public var columns: [SQLExpression]
+
+    public init(_ columns: [SQLExpression]) {
```

Comment:
A convenience init that accepts `SQLColumn` would be nice. Side note, does `RETURNING` accept qualified names? For example, can you do:

```sql
RETURNING planets.id, galaxies.id
```

If so, then `SQLColumn`. Otherwise, just `SQLIdentifier`.

Reply:
Yes it accepts qualified names and aliases as well.